### ISSUE TITLE
Modules update failing

### ISSUE THREAD

**Issue by yld** [minamijoyo/tfupdate#4]
```
# tfupdate module -v v1.2.4  git@github.com:company/terraform-modules.git//my_module . -r
failed to parse input: codebuild_service/main.tf
panic: didn't find any token of type TokenIdent
This may be caused by a bug in the hclwrite parser.
As a workaround, you can ignore this file with --ignore-path option
```

- terragrunt version v0.21.6
- Terraform v0.12.16
- tfupdate 0.3.0 (brew)

**Comment by yld** [comment]
Sorry, this is not a tfupdate, this was some errors in mine after upgrading to HCL 2.
Thanks a lot for you very useful tool.

**Comment by winmillwill** [comment]
To anyone else who hits this, the issue is that referencing a resource instance by integer index must be done with brackets now, not a dot.

valid:
```
kubernetes_namespace.my_conditional_namespaces.metadata[0].name
```

invalid:
```
kubernetes_namespace.my_conditional_namespaces.metadata.0.name
```

**Comment by minamijoyo** [comment]
@winmillwill Could you open a new issue with a minimal example?

**Comment by minamijoyo** [comment]
@winmillwill I can reproduced it and fixed in v0.3.6.
